src/App.css

```
@@ -0,0 +1,81 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+#root {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.image-wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 40px;
+}
+
+#logo {
+    width: 800px;
+}
+
+.fruit-counter-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 40px;
+}
+
+article {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin: 20px;
+    padding: 20px;
+    border: 1px solid black;
+    border-radius: 5px;
+}
+
+.red-border {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin: 20px;
+    padding: 20px;
+    border-radius: 5px;
+    border: 1px solid red;
+}
```

```suggestion
.red-border:focus-within {
    display: flex;
    flex-direction: row;
    justify-content: center;
    align-items: center;
    width: 100%;
    margin: 20px;
    padding: 20px;
    border-radius: 5px;
    border: 1px solid red;
}
```
Hiermee zorg je ervoor dat wanneer de focus bínnen het element zit, dat dán pas deze css toegepast wordt

---

Door deze change is de `onClick` functie in principe overbodig geworden

---

Dat is nieuw voor mij, weer wat geleerd!